modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/AppContext.es.js

```diff
@@ -177,17 +177,29 @@ const editFocusedCustomObjectField = ({
 	};
 };
 
+/**
+ * Get unformatted definition field
+ * @param {object} dataDefinition
+ * @param {string} fieldName
```

Comment:
Hey, this second parameter isn't a string, actually is an object that contains fieldName string

This may confuse the users that are reading JSdoc

Reply:
Yea, that is true. I'll update it